.github/workflows/e2e-test-heroku.yml

```diff
@@ -36,15 +37,23 @@ jobs:
 
       - name: creating test user
         env:
-          CONNECT_TEST_ACCOUNT_EMAIL: ${{ secrets.CONNECT_TEST_ACCOUNT_EMAIL }}
+          CONNECT_TEST_ACCOUNT_EMAIL: "taiko@e2e.test"
           CONNECT_TEST_ACCOUNT_PASSWORD: ${{ secrets.CONNECT_TEST_ACCOUNT_PASSWORD }}
           CONNECT_MANAGEMENT_URL: ${{ secrets.CONNECT_MANAGEMENT_URL }}
           CONNECT_MANAGEMENT_API_KEY: ${{ secrets.CONNECT_MANAGEMENT_API_KEY }}
         run: yarn e2e:create-test-user
 
+      - name: e2e
+        run: yarn test:e2e
+        env:
+          CONNECT_TEST_ACCOUNT_EMAIL: 'taiko_${{ github.event.deployment.sha }}@e2e.test'
+          CONNECT_TEST_ACCOUNT_PASSWORD: ${{ secrets.CONNECT_TEST_ACCOUNT_PASSWORD }}
+          CONNECT_TEST_ACCOUNT_URL: ${{ github.event.deployment.payload.web_url }}
+
       - name: removing test user
+        if: ${{ always() }}
```

Comment:
Is that condition "force" the removing user script execution, even if the e2e tests failed ?

Reply:
Yep